src/nnet3/nnet-test-utils.cc

```diff
@@ -793,7 +793,7 @@ void ComputeExampleComputationRequestSimple(
 
 static void GenerateRandomComponentConfig(std::string *component_type,
                                           std::string *config) {
-  int32 n = RandInt(0, 20);
+  int32 n = RandInt(0, 21);
```

Comment:
Yeah, I can do it.

On Wed, Nov 4, 2015 at 4:10 PM, Daniel Povey <notifications@github.com>
wrote:

> In src/nnet3/nnet-test-utils.cc
> <https://github.com/kaldi-asr/kaldi/pull/307#discussion_r43939830>:
>
> > @@ -793,7 +793,7 @@ void ComputeExampleComputationRequestSimple(
> >
> >  static void GenerateRandomComponentConfig(std::string *component_type,
> >                                            std::string *config) {
> > -  int32 n = RandInt(0, 20);
> > +  int32 n = RandInt(0, 21);
>
> @vijayaditya <https://github.com/vijayaditya>, do you have time to do a
> last pass of checking this commit?
> … <#150d4560ded80b5a_>
> On Wed, Nov 4, 2015 at 3:50 PM, hhadian <notifications@github.com> wrote:
> In src/nnet3/nnet-test-utils.cc <#307 (comment)
> <https://github.com/kaldi-asr/kaldi/pull/307#discussion_r43937518>>: > @@
> -793,7 +793,7 @@ void ComputeExampleComputationRequestSimple( > > static
> void GenerateRandomComponentConfig(std::string *component_type, >
> std::string *config) { > - int32 n = RandInt(0, 20); > + int32 n =
> RandInt(0, 21); I just removed them. — Reply to this email directly or view
> it on GitHub <https://github.com/kaldi-asr/kaldi/pull/307/files#r43937518>.
>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/307/files#r43939830>.
>